config/settings.yml

```diff
@@ -14,6 +14,9 @@ DIGIT_0: 0
 DIGIT_1000: 1000
 SIZE_400x400: [400, 400]
 range_1_5: [1, 2, 3, 4, 5]
-price_range: ["0-999", "1000-1999", "2000-2999", "3000-3999", "4000-4999", "5000-5999", "6000-6999"]
 SIZE_224x224: [224, 224]
+range_5_to_10: 5..10
```

Comment:
Những giá trị này đặt theo thực tế luôn, 5000000..10000000

Reply:
số to quá e sợ chạy lâu á